<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-01-14T00:59:15 IST" name="GOTO TRAVAL TESTING" finished-at="2024-01-14T00:59:59 IST" duration-ms="44312">
    <groups>
    </groups>
    <test started-at="2024-01-14T00:59:15 IST" name="first Test for Ticket" finished-at="2024-01-14T00:59:59 IST" duration-ms="44312">
      <class name="com.traval.qa.test.SearchHotelTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.traval.qa.test.SearchHotelTest@248e319b]" started-at="2024-01-14T00:59:15 IST" name="setup" finished-at="2024-01-14T00:59:36 IST" duration-ms="21818" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Validate_HomePage_Title()[pri:1, instance:com.traval.qa.test.SearchHotelTest@248e319b]" started-at="2024-01-14T00:59:36 IST" name="Validate_HomePage_Title" finished-at="2024-01-14T00:59:37 IST" duration-ms="868" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Validate_HomePage_Title -->
        <test-method signature="ValidataBy_AddingRequirment_InHomePage()[pri:2, instance:com.traval.qa.test.SearchHotelTest@248e319b]" started-at="2024-01-14T00:59:37 IST" name="ValidataBy_AddingRequirment_InHomePage" finished-at="2024-01-14T00:59:52 IST" duration-ms="15045" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidataBy_AddingRequirment_InHomePage -->
        <test-method signature="Validateby_Requirments_Matchingin_ListingPage_FromHomePage()[pri:3, instance:com.traval.qa.test.SearchHotelTest@248e319b]" started-at="2024-01-14T00:59:52 IST" name="Validateby_Requirments_Matchingin_ListingPage_FromHomePage" finished-at="2024-01-14T00:59:57 IST" duration-ms="4852" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Date are not match expected [Wed, Jan 17th - Thu, Jan 18th] but found [null]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Date are not match expected [Wed, Jan 17th - Thu, Jan 18th] but found [null]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at com.traval.qa.test.SearchHotelTest.Validateby_Requirments_Matchingin_ListingPage_FromHomePage(SearchHotelTest.java:64)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Validateby_Requirments_Matchingin_ListingPage_FromHomePage -->
        <test-method is-config="true" signature="Teardown()[pri:0, instance:com.traval.qa.test.SearchHotelTest@248e319b]" started-at="2024-01-14T00:59:57 IST" name="Teardown" finished-at="2024-01-14T00:59:59 IST" duration-ms="1606" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
      </class> <!-- com.traval.qa.test.SearchHotelTest -->
    </test> <!-- first Test for Ticket -->
  </suite> <!-- GOTO TRAVAL TESTING -->
</testng-results>
